src/modules/content-sqlite/content-sqlite.c

```diff
@@ -830,10 +854,18 @@ static int process_args (struct content_sqlite *ctx,
     int i;
     for (i = 0; i < argc; i++) {
         if (strstarts (argv[i], "journal_mode=")) {
+            if (!journal_mode_valid (argv[i] + 13)) {
```

Comment:
Maybe it would be good to add a `flux_error_t` argument and call out which parameter is wrong.

Reply:
good idea, although in this case I think we just do a log message, since we're not responding to a RPC